Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -81,11 +85,14 @@ def create_quam_superconducting(
             )
             machine.octaves[f"octave{i+1}"] = octave
             octave.initialize_frequency_converters()
+            # TODO: what is quam.octave?
```

Comment:
Nothing, this was before we supported multiple Octaves, maybe we should just remove this print statement